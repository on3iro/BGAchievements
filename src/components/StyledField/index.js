import styled from 'styled-components'
import { Field } from 'redux-form'

const StyledField = styled(Field)`
  display: block;
  margin: 0 0 10px 0;
  padding: 5px 10px;
  border: 1px solid ${props => props.theme.mercury};
  border-radius: 4px;
  color:${props => props.theme.text};
  outline: 0;
`

export default StyledField
