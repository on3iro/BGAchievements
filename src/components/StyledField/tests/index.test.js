import React from 'react'
import { shallow } from 'enzyme'
import { Field } from 'redux-form'

import StyledField from '../index'

describe('<StyledField />', () => {
  it('should render redux-form Field', () => {
    const renderedWrapper = shallow(<StyledField
      name='test'
      component={() => <h1>Hi</h1>}
    />)
    expect(renderedWrapper.find(Field).length).toMatchSnapshot()
  })
})
