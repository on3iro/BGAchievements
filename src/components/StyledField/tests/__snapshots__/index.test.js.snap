exports[`<StyledField /> should render redux-form Field 1`] = `1`
